### DISCUSSION TITLE
DDS Security governance file 'Work Around for check_create_participant_bug'

### DISCUSSION THREAD

**Discussion by smnrgrs** [OpenDDS/OpenDDS#4417] [Q&A]
I've been trying to enable and use DDS security. I can only get it to work between two participants on different hosts if I add into my governance file the curious section which is in tests/security/SingleParticipantWithSecurity/governance.xml:

```
<topic_rule>
    <topic_expression>Work Around for check_create_participant_bug</topic_expression>
    <enable_discovery_protection>true</enable_discovery_protection>
    <enable_liveliness_protection>true</enable_liveliness_protection>
    <enable_read_access_control>false</enable_read_access_control>
    <enable_write_access_control>false</enable_write_access_control>
    <metadata_protection_kind>ENCRYPT</metadata_protection_kind>
    <data_protection_kind>ENCRYPT</data_protection_kind>
</topic_rule>

```
I can find no bug reports relating to 'check_create_participant_bug'. Can anyone shed light on what this might be about?

Thanks

**Comment by mitza-oci** [comment]
I think it's a spec issue, not an OpenDDS bug
https://issues.omg.org/issues/task-force/DDSSEC12#issue-44156
